src/components/SQForm/SQFormRadioButtonGroup.tsx

```diff
@@ -1,13 +1,40 @@
 import React from 'react';
-import PropTypes from 'prop-types';
-import Grid from '@material-ui/core/Grid';
-import RadioGroup from '@material-ui/core/RadioGroup';
-import FormHelperText from '@material-ui/core/FormHelperText';
-import FormControl from '@material-ui/core/FormControl';
-import FormLabel from '@material-ui/core/FormLabel';
+import {
+  Grid,
+  RadioGroup,
+  RadioProps,
+  FormHelperText,
+  FormControl,
+  FormLabel,
+  GridSize
+} from '@material-ui/core';
 import SQFormRadioButtonGroupItem from './SQFormRadioButtonGroupItem';
 import {useForm} from './useForm';
 
+interface RadioButtonInputItemProps {
+  value: string | boolean | number;
+  label: string;
+  isDisabled?: boolean;
+  inputProps?: RadioProps;
```

Comment:
Sort of a nitpick, but I think we should change this to PascalCase `InputProps` to follow the pattern used in some of our other components. 
`InputProps`: props that are given to the React component
`inputProps`: props (attributes) that are given to the HTML element

This would also call for a change to `SQFormRadioButtonGroupItem`'s prop name, because it's spreading that prop directly on a Mui Radio component instead of in the `inputProps` prop

Reply:
Not sure how to go about tracing down if the inputProps should be spreading into `inputProps`
It does look like MUI's Radio does take that prop `inputProps`
This would kinda come down to how the consumers are using it, right? I mean it seems to be a path to give Radio props specific to Radio. Maybe we don't want to expose that? 

I am going to go with your suggested change, but I have no strong opinion on if we should limit this to only providing inputProps or if this should expose all of RadioProps. 

Radio's docs for convenience https://material-ui.com/api/radio/